### ISSUE TITLE
keep track of osm way id

### ISSUE THREAD

**Issue by nlehuby** [ad-freiburg/pfaedle#2]
To debug easily, it would be great to get the `way` id from OSM. Maybe in the shape_id of the GTFS or at least in the geojson `path.json` when you only compute one trip.

I will make possible to fix OSM more easily if the shape is not consistent and it could even be used to map the shapes of the transport routes in OSM (as they are OSM relations with the list of the ways in the right order)

**Comment by patrickbr** [comment]
I agree. However, `pfaedle` simplifies the input OSM graph and any 1:1 mapping between edges and OSM ways is lost. I will think about skipping these simplifications and storing the OSM way ids somewhere if a flag is set.

**Comment by nlehuby** [comment]
it would  be cool :+1:

**Comment by hbruch** [comment]
Would really be cool. This could be used to maintain OSM's route relations, also. Perhaps pfaedle could report the OSM IDs of matched stop_positions, as well.
